version: '3.7'
services:
  standalone_user_client:
    image: standalone_user_client
    ports:
      - 42055:42055
    environment:
      - ISDOCKER=cat
    networks:
      - praxisbot
    volumes:
      - "../logs:/Praxis/logs"
      - "../credentials:/Praxis/standalone_user_client/credentials"
      - "../config.py:/Praxis/standalone_user_client/config.py"
networks:
  praxisbot:
    name: praxisbot